src/utils/command.ts

```diff
@@ -0,0 +1,82 @@
+export type CommandArgumentMatchTypes = 'flag' | 'mention' | 'content' | 'rest'
+
+export interface Args {
+  name: string
+  match: CommandArgumentMatchTypes
+  type?: unknown // Still needs to be implemented
+  defaultValue?: unknown // Still needs to be implemented
+  flag?: string
+}
+
+const mentionRegex = /([0-9]{18})/g
```

Comment:
Check `src/structures/messageMentions.ts` for regexes

Reply:
I'll definitely do that. right now I just wanted it to return the userID